README.md

```diff
@@ -216,6 +216,22 @@ HELM_OPTS+="
 "
 ```
 
+### Email Connector
+
+***NB***
+In order to set email connector all the variables need to be set. If these variables are set then deployment service will use these configs as default for any applications deployed. Once these variables are configured at the deployment of chart via Helm customer won’t have the possibility to override these values from the admin app. In case you want to configure email connector variable from admin-app please dont not configure email connector during helm deployment. 
```

Comment:
I would just replace the first two statements with the below:
"In order to apply default account configuration to the email connector, all the variables need to be set. All email connectors in every applications in the cluster will share the same account."

Please consider moving this under the readme below using the description column. The main advantage would be to
increase the size of this README file also in the future when we will add more configs. And using the other readme below exclusively for properties

Reply:
I will update this, thanks